content/governance.md

```diff
@@ -34,19 +34,16 @@ New members may be proposed by any existing member by email to [prometheus-team]
 
 If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This email will also be CC'd to [prometheus-team][team] for record-keeping purposes.
 
-If they choose to accept, the following steps are taken:
-
-* Team members are added to the [GitHub organization][gh] as _Owner_.
-* Team members are added to the [team mailing list][team].
-* Team members are added to the list of team members in this document.
-* New team members are announced on the [developers mailing list][devs] by an existing team member.
+If they choose to accept, the [onboarding](#onboarding) procedure is followed.
 
 Team members may retire at any time by emailing [the team][team].
 
-Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote, does not count towards the quorum, and may not make any [editorial](#editorial-changes) changes to this and related documents any more.
```

Comment:
Why is this duplicating the removal docs? It seems very odd to call out this particular privilege of being a team member.

Reply:
To fix the race condition of them making "editorial" changes to the offboarding process.

Reply:
This is still very weird, this sentence is about the special rules of a removal vote - why is is talking about changing this doc? The governance already covers changes to it, it's not like someone can unilaterally change it.

Reply:
We now allow lazy consensus in On-/Offboarding, and in the FAQ. That's what I want to prevent.

Reply:
Should I make this more explicit? Reword? Happy?

Reply:
Leave the wording as it was, end the sentence at quorum.